schema/README.md

```diff
@@ -0,0 +1,33 @@
+# JSON schema
+
+## Overview
+
+This directory contains the [JSON Schema](http://json-schema.org/) for
+validating the `config.json` of this container runtime specification.
+
+The layout of the files is as follows:
+* schema.json - the primary entrypoint for the whole schema document
+* schema-linux.json - this schema is for the Linux specific sub structure
+* defs.json - definitions for general types
+* defs-linux.json - definitions for Linux specific types
+* validate.go - validation utility source code
+ 
+
+## Utility
+
+There is also included a simple utility for facilitating validation of a
+`config.json`. To build it:
+
+```bash
+export GOPATH=`mktemp -d`
+go get ./...
+go build .
+rm -rf $GOPATH
+```
+
+Then use it like:
+
+```bash
+./schema ./schema.json <yourpath>/config.json
```

Comment:
again, fair. It is pedantic. Even a tool like `godoc` will search $GOPATH unless the args have a `./` or explicit path. I'll change though if it seems unnatural to read ...